ClrMD.Extensions/ClrDynamic.cs

```diff
@@ -726,13 +815,13 @@ public static string GetSimpleValueString(ClrDynamic obj)
 
             private static byte[] ReadBuffer(ClrHeap heap, ulong address, int length)
             {
-                byte[] buffer = new byte[length];
-                int byteRead = heap.ReadMemory(address, buffer, 0, buffer.Length);
+                Span<byte> buffer = new byte[length];
+                int byteRead = heap.Runtime.DataTarget.DataReader.Read(address, buffer);
 
                 if (byteRead != length)
                     throw new InvalidOperationException(string.Format("Expected to read {0} bytes and actually read {1}", length, byteRead));
 
-                return buffer;
+                return buffer.ToArray();
```

Comment:
I think this is creating a new array, it would be better to declare buffer as `byte[]`, there will be an implicit conversion to Span<byte> when calling `DataReader.Read`